tests/test_wave_03.py

```diff
@@ -1,22 +1,23 @@
 import pytest
 from viewing_party.party import *
 from tests.test_constants import *
-
-@pytest.mark.skip()
+# TEST ONE - PASSED
+# @pytest.mark.skip()
 def test_my_unique_movies():
     # Arrange
     amandas_data = clean_wave_3_data()
 
     # Act
     amandas_unique_movies = get_unique_watched(amandas_data)
+    print(amandas_unique_movies)
```

Comment:
Style: It's a best practice to clean debug `print`s up before you turn things in for review.

Reply:
Oops forgot about that one! Thank you, Kaida!!